package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gavel2html",
-  "version": "0.0.0-semantically-released",
-  "description": "Render Gavel.js output to HTML diff",
+  "version": "0.2.0-unstable.1",
```

Comment:
As a next step, I'd bump this in a separate PR to `0.2.0-unstable.2` or `0.2.1`, release, and upgrade in the core app, so we see this is really the code we think it is 🙈

Reply:
Linked the package and tried in the core app. Confirms working without changes.